### ISSUE TITLE
When pkg-config is missing, build fails in a strange way

### ISSUE THREAD

**Issue by mojca** [msharov/snownews#35]
The code should probably check whether `pkg-config` exists before calling it:
```
Running configure ...
./configure: line 209: --exists: command not found
```

**Comment by maxice8** [comment]
This most likely is the cause of #41 (besides the original poster not having libxml2-devel installed)

**Comment by msharov** [comment]
Fixed by commit 268ef84660c03df29e8f155c3570092add1596cd
